runscanner/src/main/java/uk/ac/bbsrc/tgac/miso/runscanner/processors/DefaultIllumina.java

```diff
@@ -44,6 +48,11 @@
 
   private static final XPathExpression RUN_COMPLETION_STATUS_EXPRESSION = compileXPath("//CompletionStatus")[0];
 
+  private static final Pattern COMMA = Pattern.compile(",");
+
+  private static final Predicate<String> BCL_FILENAME = Pattern.compile("^s_[0-9]*_[0-9]*\\.bcl(\\.gz)?").asPredicate();
+  private static final Predicate<String> BCL_BGZF_FILENAME = Pattern.compile("^[0-9]*\\.bcl\\.bgzf").asPredicate();
+
   public static DefaultIllumina create(Builder builder, ObjectNode parameters) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused method parameter "parameters". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1172)